wix/Build.OpenJDK_generic.cmd

```diff
@@ -40,6 +40,11 @@ IF NOT DEFINED PRODUCT_UPDATE_INFO_LINK SET PRODUCT_UPDATE_INFO_LINK=https://ado
 IF NOT DEFINED WIX_HEAT_PATH SET WIX_HEAT_PATH=.\Resources\heat_dir\heat.exe
 IF NOT DEFINED WIX_VERSION SET WIX_VERSION=5.0.0
 
+REM default windows_SDK version
+REM See folder e.g. "C:\Program Files (x86)\Windows Kits\[10]\bin\[10.0.22621.0]\!PLATFORM!"
+IF NOT DEFINED WIN_SDK_MAJOR_VERSION SET WIN_SDK_MAJOR_VERSION=10
```

Comment:
Maybe define those CONST values up above and refer to them that way?

Reply:
Not fully sure what this means. Are you asking me to define these constants and to move them up a few lines? These were previously defined as constants in the script below, I just moved them up because I wanted the user to have the ability to change the used windows_sdk without requiring a code change

Reply:
OK, yeah fine, leave it as is.